### DISCUSSION TITLE
A question when running datasetbuilder

### DISCUSSION THREAD

**Discussion by Anopaul** [tongzhugroup/mddatasetbuilder#144] [Q&A]
Hello everyone.

I encountered a problem when using the 'datasetbuilder' command to try to build a dataset for n-C12 pyrolysis. As shown in the figures below:

Using the 'jobs' command shows that my task is still running. However, using the 'top' command, the task does not invoke the computation resources, and the log file is no longer updated. I guess it's stuck at what point?

I wanna ask what may cause this problem.

Besides, the mddatasetbuilder version is  1.3.4. The input file is the trajectory file of a 3000K, 10000 steps, 0.1fs. NVT simulation running in Lammps.
![屏幕截图 2022-08-12 102139](https://user-images.githubusercontent.com/108926439/184279741-b5c5b8d2-7af5-4e0b-92dc-57a896076ae3.png)
![屏幕截图 2022-08-12 102201](https://user-images.githubusercontent.com/108926439/184279751-e96ce4cf-8f75-494f-bd04-318a2f9e108c.png)
![屏幕截图 2022-08-12 102037](https://user-images.githubusercontent.com/108926439/184279758-f9cf77ca-f2b8-4a7d-8e2c-03d7d81b7099.png)

**Comment by njzjz** [comment]
You can check if something was killed.
https://stackoverflow.com/q/726690/9567349

**Comment by KangBaoBBS** [comment]
I meet the same problem. Can you share it?

**Reply by Anopaul** [reply]
My previous solution was to use another version of mddatasetbuilder.

**Reply by KangBaoBBS** [reply]
Thank you for your reply! I set the size of the data by the "-s 100". Then it works.
